### ISSUE TITLE
Lisp version of the compiler

### ISSUE THREAD

**Issue by eschaton** [jbush001/LispMicrocontroller#1]
It’d be cool to have a version of the compiler that’s itself written in Lisp.

It might even be cool to have an interpreter written in something like Common Lisp, and then have the compiler written in that. (With more and writable storage, that would make it feasible to self-host.)

**Comment by jbush001** [comment]
Yes, writing the compiler in Lisp seems sorta obligatory at some point. :)
